ncf_pos/static/src/js/models.js

```diff
@@ -310,11 +310,14 @@ odoo.define('ncf_pos.models', function (require) {
         get_orders_from_server: function () {
             var self = this;
             var day_limit = this.config.order_loading_options === 'n_days' ? this.config.number_of_days : 0;
+            var max_id = Math.max(..._.map(this.db.pos_all_orders, function(o){
```

Comment:
Que?

Reply:
ok, it's ES6 feature. You must check browser support before using it

Reply:
So, will this work in Chrome, Opera and Firefox?

Reply:
according to this 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax
it doesnt work only in the Internet Explorer